frontend/projects/upgrade/src/app/app.module.ts

```diff
@@ -29,6 +29,7 @@ export const getEnvironmentConfig = (http: HttpClient, env: Environment) => {
       .then((config: RuntimeEnvironmentConfig) => {
         env.apiBaseUrl = config.endpointApi || config.apiBaseUrl;
         env.googleClientId = config.gapiClientId || config.googleClientId;
+        env.featureFlagNavToggle = config.featureFlagNavToggle ?? env.featureFlagNavToggle ?? false;
```

Comment:
Do we need this line?

Reply:
Yes if we want to be able to toggle this flag at runtime in a particular environment without rebuild/redeploys, this was the only way I can think to do that. That will override the build config if present, and be false if it is not present in build config either.